application-forum-ui/src/main/resources/ForumCode/AnswerSheet.xml

```diff
@@ -53,6 +53,21 @@
         &lt;/dl&gt; 
       &lt;/div&gt;
     {{/html}}
+    #elseif ($request.commentEditForm == 'true')
+    {{html}}
+      #set ($commentEditForm = "#template('commentsinline.vm')")
+      #if (!$commentEditForm.contains('&lt;textarea/&gt;'))
```

Comment:
Missing comment to explain **why** the textarea could be missing. What does it mean if the text area is present and what does it mean if it's not?